src/js/repl.js

```diff
@@ -42,10 +42,17 @@ export function prompt(rl: readline$Interface,
 }
 
 export function processLine(rl: readline$Interface, line: string): void {
+  // $FlowIssue - use of rl.output
+  const isMainRepl = rl.output === process.stdout;
```

Comment:
not sure what the perf cost of making this comparison on every input is, but I wouldn't mind having an additional `isMainREPL` function parameter to which we pass `false` in the socketREPL module.